### DISCUSSION TITLE
How to programmatically add a thumbs slider to another slider when both sliders have already been initialized?

### DISCUSSION THREAD

**Discussion by Chaitanya-Raj** [nolimits4web/swiper#6869] [Q&A]
According to the docs, the way to add a thumbs slider to another slider is to provide the swiperInstance of the thumb slider to the params while initializing the slider.

How can a thumbs slider be added to a slider when both of the sliders have already been initialized?

I have tried following the same method as linking the two sliders i.e. slider1.thumbs.swiper = slider2.
When doing this, the sliders are linked together, but clicking on the thumbs slider does not move either the active slide or the other slider.

Link to a CodeSandbox showing the issue: https://codesandbox.io/p/sandbox/swiper-thumbs-gallery-forked-kqflj2?file=%2Findex.html%3A199%2C36

**Comment by cernymatej** [comment]
You can add it to the main swiper at runtime like this:
```ts
mainSwiperInstance.params.thumbs = {
  swiper: thumbsSwiperInstance
}
```

Hope this helps!
